### PR TITLE
add more traces

### PR DESCRIPTION
**Pull request by findmyway**

### CODE REVIEW

File: src/trajectory.jl [hunk 1]
```diff
@@ -53,6 +56,16 @@ Base.@kwdef struct Trajectory{C,S,T}
     end
 end
 
+TrajectoryStyle(::Trajectory) = SyncTrajectoryStyle()
+TrajectoryStyle(::Trajectory{<:Any,<:Any,<:AsyncInsertSampleRatioControler}) = AsyncTrajectoryStyle()
+
+Base.bind(::Trajectory, ::Task) = nothing
+
+function Base.bind(t::Trajectory{<:Any,<:Any,<:AsyncInsertSampleRatioControler}, tassk)
```
Comment (HenriDeh) [inline src/trajectory.jl]: Typo here : tassk -> task

Comment (findmyway) [reply src/trajectory.jl]: Fixed, thanks!

**Comment by HenriDeh** [comment]
I was wondering if adding Traces that incorporate :action_log_prob into the defaults would not be appropriate ? Many algorithms use it so it should be useful to standardize the way they work I think.

**Comment by findmyway** [comment]
> way

Sure, we can definitely add them

**Pull request closed without merging**